tests/expected/intrinsics/copy-nonoverlapping/copy-overlapping/expected

```diff
@@ -0,0 +1 @@
+memcpy src/dst overlap
```

Comment:
Is this a CBMC message? I was wondering if we change it to say copy_nonoverlap here. Do we use memcpy anywhere else?

Reply:
Yes, this is a CBMC message so there's not much we can do. We use `memcpy` in our `__rust_realloc` C implementation but we don't generate calls to `memcpy` anywhere else.